app/build.gradle

```diff
@@ -24,4 +24,6 @@ dependencies {
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.3.0'
     compile 'com.android.support:design:23.3.0'
+    compile 'com.android.support:support-annotations:25.0.0'
+    compile 'com.google.guava:guava:20.0'
```

Comment:
The Guava project contains several of Google's core libraries that we rely on in our Java-based projects.

You can check this [link](https://github.com/google/guava/wiki)